src/Neo.VM/ExecutionEngineLimits.cs

```diff
@@ -36,12 +36,12 @@ public sealed record ExecutionEngineLimits
         /// <summary>
         /// The maximum size of an item in the VM.
         /// </summary>
-        public uint MaxItemSize { get; init; } = 1024 * 1024;
+        public uint MaxItemSize { get; init; } = ushort.MaxValue;
 
         /// <summary>
         /// The largest comparable size. If a <see cref="Types.ByteString"/> or <see cref="Types.Struct"/> exceeds this size, comparison operations on it cannot be performed in the VM.
         /// </summary>
-        public uint MaxComparableSize { get; init; } = 65536;
+        public uint MaxComparableSize { get; init; } = ushort.MaxValue;
```

Comment:
but what is the difference to change `65536` to `65535` here? is 1 byte really hurt?